src/geometrics/Circle.java

```diff
@@ -21,8 +20,7 @@ public double perimeter() {
     }
 
     public void display() {
-        super.display();
-        System.out.print("; Radius= " + getRadius());
+        System.out.print("Radius= " + getRadius());
     }
 
     public String name() {
```

Comment:
It's better to rename this method to getName or something like this.
Also you could get a name using this.getClass().getSimpleName() instead of returning hardcoded "Circle" string. You even can transfer method with such realization to your parent class (Shape) .